sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/Statsbeat.cs

```diff
@@ -43,6 +47,48 @@ internal static class Statsbeat
 
         internal static MeterProvider s_attachStatsBeatMeterProvider;
 
+        internal static Regex s_endpoint_pattern = new("^https?://(?:www\\.)?([^/.-]+)");
+
+        internal static readonly HashSet<string> EU_Endpoints = new()
+        { "francecentral", "francesouth", "northeurope", "norwayeast", "norwaywest", "swedencentral", "switzerlandnorth", "switzerlandwest", "uksouth", "ukwest", "westeurope", };
+
+        internal static readonly HashSet<string> Non_EU_Endpoints = new()
+        {
+            "australiacentral",
+            "australiacentral2",
+            "australiaeast",
+            "australiasoutheast",
+            "brazilsouth",
+            "brazilsoutheast",
+            "canadacentral",
+            "canadaeast",
+            "centralindia",
+            "centralus",
+            "chinaeast2",
+            "chinaeast3",
+            "chinanorth3",
+            "eastasia",
+            "eastus",
+            "eastus2",
+            "japaneast",
+            "japanwest",
+            "jioindiacentral",
+            "jioindiawest",
+            "koreacentral",
+            "koreasouth",
+            "northcentralus",
+            "qatarcentral",
+            "southafricanorth",
+            "southcentralus",
+            "southeastasia",
+            "southindia",
+            "uaecentral",
+            "uaenorth",
+            "westus",
+            "westus2",
+            "westus3",
+        };
```

Comment:
nit: The EU list is one line, and the non-EU list is multiline.
I would recommend formatting these the same to make them easier to read and maintain.

Second, consider moving these to a new file: Statsbeat/Constants.cs

Reply:
will do as a follow up.